.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ ci:
 
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
+    rev: v4.0.1
```

Comment:
No need to do this automatically, I added a CI, it'll take care of this file automatically from now on